workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/ValidatorMessageHandler.java

```diff
@@ -40,6 +41,10 @@ public static void handleErrMsg(String exceptionMsg, boolean isInvalid, boolean
         }
         WorkflowLogger.logPersistedActivities();
         if (isFailed) {
+            if (WorkflowLogger.isDisabled() && exceptionMsg != null && !exceptionMsg.isEmpty()) {
```

Comment:
exceptionMsg != null && !exceptionMsg.isEmpty(), can be replaced using StringUtils